src/Intuition.php

```diff
@@ -1360,8 +1360,10 @@ protected function initLangSelect( $option ) {
 					continue;
 				}
 
-				while ( ( $n = strstr( $acceptLang, '-' ) ) !== false ) {
```

Comment:
Oops, I guess this was previously intended to be `strpos()` - which is similar, but returns an integer indeed. Anyhow, this seemingly never worked right. Thanks!